frontend/public/components/utils/units.js

```diff
@@ -138,15 +138,58 @@ const humanize = units.humanize = (value, typeName, useRound = false) => {
 };
 
 export const humanizeMem = v => humanize(v, 'binaryBytes', true).string;
-export const humanizeCPU = v => humanize(v, 'numeric', true).string;
 export const humanizeNumber = v => humanize(v, 'numeric', true).string;
+export const humanizeCPU = v => humanize(v, 'numeric', true).string;
+export const dehumanizeCPU = v => {
+  const _dehumanizeCPU = _.spread((value, unit) => {
```

Comment:
Prefer es6 syntax to `_.spread`:

```suggestion
  const _dehumanizeCPU = ([value, unit]) => {
```